Source/Dronecode-SDK-Swift/Dronecode-SDK-Swift/Camera.swift

```diff
@@ -2,7 +2,93 @@ import Foundation
 import SwiftGRPC
 import RxSwift
 
+// MARK: - CameraResult
```

Comment:
The file has suddenly the executable bit set, see `100644 → 100755`. Please fix it using `chmod -x Camera.swift`.